### PR TITLE
updated troposphere to version 3.0.2

### PR DESCRIPTION
**Pull request by shimwell**
As discussing in issue #1 

This is a minor update that gets the poetry install working again.

### CODE REVIEW

**Comment by shimwell** [comment]
Looks like the solution is not quite as simple as this, updating to troposphere version 3.0.2 results in new errors in the final deploy stage

```python
    self.cf_template.add_description("Automatically generated with Zappa")
AttributeError: 'Template' object has no attribute 'add_description'
```

results in a new error discussed here https://stackoverflow.com/questions/68391621/zappa-deploy-fails-with-attributeerror-template-object-has-no-attribute-add

File: poetry.lock [hunk 1]
```diff
@@ -641,7 +641,7 @@ tqdm = [
     {file = "tqdm-4.61.1.tar.gz", hash = "sha256:24be966933e942be5f074c29755a95b315c69a91f839a29139bf26ffffe2d3fd"},
 ]
 troposphere = [
-    {file = "troposphere-2.7.1.tar.gz", hash = "sha256:ea2e5f2f82c224eaa1414a008b1939aae124c3e3e1dd993301968f155b333bd7"},
+    {file = "troposphere-3.0.2.tar.gz", hash = "sha256:acde51e47294bdb2bce4f0be4320ac12dbd3c6a2783895779ddb94cfdc0badaf"},
```
Comment (ian-whitestone) [inline poetry.lock:644]: Thanks for opening this! Can we instead try doing `poetry add zappa==0.54.0`? That will also bump the troposphere dependency

**Comment by ian-whitestone** [comment]
> Looks like the solution is not quite as simple as this, updating to troposphere version 3.0.2 results in new errors in the final deploy stage
> 
> ```python
>     self.cf_template.add_description("Automatically generated with Zappa")
> AttributeError: 'Template' object has no attribute 'add_description'
> ```
> 
> results in a new error discussed here https://stackoverflow.com/questions/68391621/zappa-deploy-fails-with-attributeerror-template-object-has-no-attribute-add

yeah so the new Zappa release, 0.54.0, implements the fix for this

File: poetry.lock (hunk 1 above)
Comment (shimwell) [reply poetry.lock:644]: entering ```poetry add zappa==0.54.0``` into the terminal at the start appears to have solved the troposphere difficulties.

Comment (shimwell) [reply poetry.lock:644]: Perhaps the only line to change is to bump this line up to 0.54.0 then
https://github.com/ian-whitestone/zappa-serverless-docker/blob/f7e2696830f2bcfa4cb2654f3aa84fb3d02a8f5c/pyproject.toml#L9

Comment (ian-whitestone) [reply poetry.lock:644]: When you run that command I believe it should automatically update both `pyproject.toml` and the `poetry.lock` files. Are you able to add those modified files to this PR? If not, I will do that tonight!

**Comment by shimwell** [comment]
closed in favor of #3

**Pull request closed without merging**